device/ui/hud.tscn

```diff
@@ -0,0 +1,50 @@
+[gd_scene load_steps=4 format=1]
+
+[ext_resource path="res://globals/hud.gd" type="Script" id=1]
+[ext_resource path="res://ui/inventory.scn" type="PackedScene" id=2]
+[ext_resource path="res://ui/verb_menu.scn" type="PackedScene" id=3]
```

Comment:
This dependency needs to be fixed.

`[ext_resource path="res://ui/verb_menu.tscn" type="PackedScene" id=3]`